### DISCUSSION TITLE
Port change

### DISCUSSION THREAD

**Discussion by StarNumber12046** [coder/code-server#3279] [General] [answered]
How to change the port of the webserver?

**Comment by jsjoeio** [comment, marked as answer]
You can change ports using the `--bind-addr` flag or `$PORT`.

Let' say you want to use `3000`. Either of these two options would work:
```
# option 1, assuming you're serving on localhost
code-server --bind-addr 127.0.0.1:3000

# option 2
PORT=3000 code-server
```

For the future, you can run `code-server --help` to see supported flags.

**Reply by StarNumber12046** [reply]
ok

**Reply by shawywang** [reply]
It should be like this :`./code-server --host "0.0.0.0" --port=8084`
I'm getting on this now and It works
